Source/Transforms/LMCWindowFunctions.hh

```diff
@@ -0,0 +1,56 @@
+/*
+ * LMCWindowFunctions.hh
+ *
+ *  Created on: Jan 10, 2022
+ *      Author: A. B. Telles
+ */
+
+#ifndef LMCWINDOWFUNCTIONS_HH_
+#define LMCWINDOWFUNCTIONS_HH_
+
+#include "param.hh"
+
+namespace locust
+{
+    /*!
+     @class WindowFunctions
+     @author A. B. Telles
+     @brief Class to create and apply window functions
+     @details
+     Currently being used for the complex FFT class. Can be expanded to other things such as LPF in the future.
+     
+     Available configuration options:
+     
+     */
+    class WindowFunctions
+    {
+        
+    public:
+        WindowFunctions();
+        virtual ~WindowFunctions();
+        bool Configure( const scarab::param_node& aNode);
+        bool GenerateWindowFunction(int windowsize, int prewindowbins, int signalsize);
+        bool SetupWindow(std::string windowname, double windowparam);
+        std::vector<double>* GetWindowFunction();
```

Comment:
Here are a couple of changes to return vector the vector is an slighlty cleaner way. I will put comments at other places to account for this change. 
Here, I'd change this to 
```const std::vector<double>* GetWindowFunction() const;```
This will return a vector that can't be modified outside of this class.

Reply:
I couldn't get the const to compile ... could you take a look at my changes and see?

Reply:
Hi Arina, I added a couple more comments. Let me know if that works.